db/column_family.cc

```diff
@@ -985,11 +985,11 @@ Compaction* ColumnFamilyData::CompactRange(
     int output_level, uint32_t output_path_id, uint32_t max_subcompactions,
     const InternalKey* begin, const InternalKey* end,
     InternalKey** compaction_end, bool* conflict,
-    uint64_t max_sst_file_number) {
+    uint64_t next_file_number) {
```

Comment:
Still not very easily understood to me. Imagine a user reads function CompactRange(), and sees a function named `next_file_number`, what he would think about it is?

Maybe something like `max_file_num_to_ignore`, or something like that?